test/actions/deploy_action.js

```diff
@@ -97,12 +97,12 @@ describe('Deploy Actions', () => {
   describe('deployInitial', () => {
     it('passes contract jsons to the deployer together with already deployed genesis addresses', async () => {
       expect(await deployActions.deployInitial()).to.equal(exampleDeployResult);
-      expect(mockDeployer.deploy).to.be.calledOnceWith(contractJsons, genesisContracts);
+      expect(mockDeployer.deploy).to.be.calledOnceWith(contractJsons, genesisContracts, [], {multiplexer: {owner: defaultAddress}}, sinon.match(/^\d+\.\d+\.\d+$/));
```

Comment:
I don't like the regex repeating here. Either:
1) Extract the matcher to a function or regex to constant
2) Or just include version from package as in production code and verify it's equal.